### PR TITLE
feat: compare light vs full ecModel

### PR DESCRIPTION
**Pull request by edkerk**
### Main improvements in this PR:
- chore:
  - full rerun `full_ecModel` tutorial, resulting in minor changes in various files.
- feat:
  - compare light vs full ecModel as Step 30 in `full_ecModel`.
- fix:
  - `makeEcModel` correct metCharges.
  - `sensitivityTuning` as last, report final growth rate, not penultimate growth rate.

**I hereby confirm that I have:**

- [x] Selected `develop` as a target branch (top left drop-down menu)

### CODE REVIEW

File: tutorials/full_ecModel/protocol.m [hunk 1]
```diff
@@ -95,7 +95,7 @@
 [ecModel, foundComplex, proposedComplex] = applyComplexData(ecModel);
```
Comment (Yu-sysbio) [inline tutorials/full_ecModel/protocol.m:95]: Running Line 95 got:

> Conversion to cell from char is not possible.
> 
> Error in applyComplexData (line 75)
>     complexProts(lastProt+1:lastProt+numel(newProts))=newProts;

Comment (edkerk) [reply tutorials/full_ecModel/protocol.m:95]: I cannot replicate this. If this error persists, can you try the following?
```
complexInfo = getComplexData();
[ecModel, foundComplex, proposedComplex] = applyComplexData(ecModel);
```
If you still get the error, please run `save('complex.mat','ecModel','complexInfo')` and attach `complex.mat` to your comment.

Comment (Yu-sysbio) [reply tutorials/full_ecModel/protocol.m:95]: Weird that today I did not get the error by doing the same thing as yesterday. Now there is no problem in this line.

**Review by Yu-sysbio — APPROVED** [review]
Looks good. No error occurs when running full_ecModel/protocol.m

**Pull request merged**